### ISSUE TITLE
8bitdo Micro Smooth Scrolling Issue

### ISSUE THREAD

**Issue by Ammar360** [roxgib/anki-contanki#119]
Hello,

I've been using the 8bitdo micro with the Contanki addon on my anki v24.06.3

I've tried to use the smooth scrolling function on on answer sections on my cloze cards (I have multiple Sketchy pics I want to scroll through and the traditional scroll button goes too far). However, the smooth scrolling is continuous and doesn't stop when I release the button. Even worse, if I try to scroll back up with the traditional scroll button, once I scroll up, the smooth scroll with continuously go and won't stop until it reaches the bottom, even though I haven't activated the button. When I go to another card and show the answer, it will again continue scrolling without button activation.

I'd appreciate any and all help with this matter. If the smooth scrolling isn't fixable, I'd appreciate guidance on how to reduce the amount of scrolling each button press does with the regular scroll button.

I copied this from another user on reddit who is experiencing the same exact issue as well as of 22 days ago, so it appears to be occurring to multiple people.

**Comment by noobnooob197** [comment]
Smooth scrolling has always been unfortunately kind of broken... I use standard scroll. You can also adjust the scroll speed underneath options on the main page. I find about 4 or 5 is perfect.
It would also probably help you alot is used the anking note type add on to customise which additional fields you want to see every time and which you don't, so you don't always have to scroll through everything.

**Comment by chelof2020** [comment]
Yeah, still does not work and it's 2026. Hope the fix it soon (wish they maybe fix it this year)
